### PR TITLE
Gossipping girls gone wild

### PR DESCRIPTION
**Pull request by lars4509**
Expanded current GG models with 5 and 6 players, and updated queries as well

### CODE REVIEW

File: benches/benchmark_solver.rs [hunk 1]
```diff
@@ -423,86 +423,86 @@ bench_lcgs!(
 // gossiping girls
 bench_lcgs!(
     gg1_circular,
```
Comment (NicEastvillage) [inline benches/benchmark_solver.rs:425]: I renamed these to `ggN_circularQ` where `N` is the number of girls and `Q` is the query index. That way you can add `gg5_circularQ` to the bench as well.

**Review by NicEastvillage — CHANGES_REQUESTED** [review]
Found this

**Review by falkecarlsen — APPROVED** [review]
:shipit:

File: benches/benchmark_solver.rs [hunk 2]
```diff
@@ -2387,20 +2387,20 @@ criterion_group!(
...
-    gg6_total_threads,
-    gg7_total_threads, // gg_total benches takes avg 69s a piece (nice)
+    gg4_circular1_threads,
+    gg4_circular2_threads,
+    gg4_circular3_threads,
+    gg4_circular4_threads,
+    gg4_circular5_threads,
+    gg4_circular6_threads,
+    gg4_circular7_threads,
+    gg4_total1_threads,
```
Comment (NicEastvillage) [inline benches/benchmark_solver.rs:2397]: Also add `gg5_circ` to bench

**Review by NicEastvillage — CHANGES_REQUESTED** [review]

File: benches/benchmark_solver.rs (hunk 2 above)
Comment (NicEastvillage) [reply benches/benchmark_solver.rs:2397]: Nvm

**Review by NicEastvillage — APPROVED** [review]

**Pull request merged**